semeio/communication/semeio_script.py

```diff
@@ -24,7 +24,7 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 
 class _ReportHandler(BufferingHandler):
     def __init__(self, output_dir, thread_id):
-        super(_ReportHandler, self).__init__(1)
+        super().__init__(1)
```

Comment:
These are all completely python2 backwards compatibility that is removed, correct? Let's put that in a single commit and mark it as such - will help us in the long run! :)